.github/workflows/hasura-preview.yml

```diff
@@ -2,6 +2,7 @@ name: 'hasura-cloud-preview'
 on:
   pull_request:
     types: [opened , reopened, synchronize]
+    if: ${{ !github.event.pull_request.head.repo.fork }}
```

Comment:
would be better to check if the head repo org/name matches the current repo with `github.event.pull_request.head.repo.full_name == github.repository`

Reply:
Mostly because its more explicit in what its checking (that the repo the PR branch is from exactly matches our repo) vs this just checking if the incoming branch is a fork. There could be scenarios where someone creates a copy of our repo without forking it and make PRs from that.